### PR TITLE
Update help for recent changes

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html [hunk 1]
```diff
@@ -8,28 +8,65 @@
...
+<H3>Use ajax spider</H3>
+
+The ajax spider explores the application by launching the browser you have chosen and clicking on the links it finds. 
+Is is slower than the traditional spider but handles JavaScript well.<br>
+This option is only shown if the ajax spider add-on is installed. 
+If it is not available then you can download and install it for free from the ZAP Marketplace.
+
+<H2>Manual Explore</H2>
+
+The Manual Explores screen allows you to launch the browser of your choice so that you can explore your application while proxying through ZAP.
```
Comment (kingthorin) [inline addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html]: Explores doesn't need to be plural.

Note it ignores cert issues and disables browser security controls?

File: addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html [hunk 2]
```diff
@@ -8,28 +8,65 @@
...
+
+<H3>URL to explore</H3>
+
+This is the URL of the application you wish to explore, starting with 'http://' or 'https://'.<br>
+ZAP must be able to access this URL.
+
+<H3>Enable HUD</H3>
+
+The ZAP Heads Up Display (HUD) brings all of the essential ZAP functionality into your browser. 
+It is ideal for people new to web security and also allows experienced penetration testers to focus on an applications
```
Comment (kingthorin) [inline addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html]: Applications should be possessive `aaplication's`

**Comment by psiinon** [comment]
Fixed those 2 :)

File: addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html [hunk 3]
```diff
@@ -8,28 +8,65 @@
...
+
+This is the URL of the application you wish to explore, starting with 'http://' or 'https://'.<br>
+ZAP must be able to access this URL.
+
+<H3>Enable HUD</H3>
+
+The ZAP Heads Up Display (HUD) brings all of the essential ZAP functionality into your browser. 
+It is ideal for people new to web security and also allows experienced penetration testers to focus on an application's 
+features while providing key security information and functionality. <br>
+This option is only shown if the HUD add-on is installed.
```
Comment (thc202) [inline addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html]: `This option is disabled if...`?

Comment (psiinon) [reply addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html]: done :)

**Comment by thc202** [comment]
Thank you!

**Pull request merged**